File: 1221-Split a String in Balanced Strings-分割平衡字符串/1221-Split a String in Balanced Strings.java
class Solution {
    public int balancedStringSplit(String s) {
        int mount = 0;
        int count = 0;
        for(int i = 0; i < s.length(); i++){
          if(s.charAt(i) == 'L'){
            mount--;
          }else if(s.charAt(i) == 'R'){
            mount++;
          }
          if(mount == 0){
            count++;
          }
        }
        return count;
    }
}

File: 1221-Split a String in Balanced Strings-分割平衡字符串/1221-Split a String in Balanced Strings.js
/**
 * @param {string} s
 * @return {number}
 */
var balancedStringSplit = function(s) {
  var mount = 0
  var count = 0
  for(let i = 0; i< s.length; i++){
      if(s[i] === 'L') mount--
      else if(s[i] === 'R') mount++
      if(mount === 0) count++
  }
  return count
};
